src/Native/Silk.NET.Vulkan.SwiftShader.Native/Silk.NET.Vulkan.SwiftShader.Native.csproj

```diff
@@ -12,19 +14,7 @@
     <PackageProjectUrl>https://github.com/dotnet/Silk.NET</PackageProjectUrl>
     <Description>Multi-platform native library for SwiftShader.</Description>
     <RepositoryType>git</RepositoryType>
-    <RepositoryUrl>https://swiftshader.googlesource.com/SwiftShader/</RepositoryUrl>
```

Comment:
But isn't this upstream?

Reply:
I thought it would be better for auditing purposes to point to the repository that we actually produce artifacts from even if it's a mirror.

Reply:
ok fine